sdk/graphql/forta.ts

```diff
@@ -0,0 +1,127 @@
+import { Alert } from '../alert'
+
+export const FORTA_GRAPHQL_URL = "https://api.forta.network/graphql";
```

Comment:
Would it be possible to inject this? I saw that `index.ts` in the sdk imports the `awilixConfigureContainer`, but didn't see anything in the sdk using it.

Reply:
Could we also create a separate container on the sdk itself?

Reply:
the plan is to break the sdk and cli out into their own npm packages. there would also be a core package which would contain most of the functionality